### ISSUE TITLE
Fehlermeldung beim Einloggen in Lovelace

### ISSUE THREAD

**Issue by andiweli** [ioBroker/ioBroker.cloud#372]
**Describe the bug**  
Error "Cannot request web pages "http://10.1.0.10:8082/browser_mod.js": File not found: {}" erscheint im Log, jedesmal wenn man sich in Lovelace über den Pro Account von extern einloggt (zb. in der iOS App).

**To Reproduce**  
Einloggen von extern mit Pro Account

**Expected behavior**  
Keine Fehlermeldungen :)

**Screenshots & Logfiles**  
`Cannot request web pages "http://10.1.0.10:8082/browser_mod.js": File not found: {}`

**Versions:**  
 - Adapter version: 4.4.1
 - JS-Controller version: 4.0.24
 - Node version: 18.17.1
 - Operating system: Raspberry OS Bullseye

**Comment by Maschine20** [comment]
Ich habe selbiges Problem.

**Comment by Grmlrckr** [comment]
Gleiches Problem hier

**Comment by Apollon77** [comment]
Welcher lovelace version?

**Comment by Grmlrckr** [comment]
v3.0.1

**Comment by Apollon77** [comment]
@Garfonso any idea?

**Comment by Garfonso** [comment]
Ja, das ist ein Problem in 3.0.1 und dem Zugriff über die ioBroker Cloud -> führt dazu, dass Browser, die sich auf diese Weise verbinden nicht ferngesteuert werden können.
Das ist in Version 4.x behoben (die aber gerade erst ins latest gekommen ist -> noch etwas Geduld, aber mittlerweile scheinen die gröbsten Fehler raus).
